services/analysis-cached.js

```diff
@@ -5,17 +5,21 @@ import DATASETS from 'data/analysis-datasets.json';
 import snakeCase from 'lodash/snakeCase';
 import moment from 'moment';
 
-import { GFW_API } from 'utils/apis';
+import { GFW_DATA_API, GFW_STAGING_DATA_API } from 'utils/apis';
+
+const ENVIRONMENT = process.env.NEXT_PUBLIC_FEATURE_ENV;
+const GFW_API =
```

Comment:
could you add a check here to make sure we are using prod and not staging on preprod env? feels more semantic unless this is what you want

Reply:
will do!